content/docs/SUSHI/installation/_index.md

```diff
@@ -27,13 +27,9 @@ To install SUSHI, open up a command prompt and type the following command:
 Check the installation via the command below:
 
 ```shell
-{{< terminal >}} sushi --help
+{{< terminal >}} sushi help
```

Comment:
Oh, interesting, because `sushi --help` works too!

And apparently you can do `sushi help init` too? I've always done `sushi init --help`.

Look at how flexible we are! (OK, how flexible commander is).

Reply:
I relearned this today too!